### PR TITLE
Completed - diagnostic

### PR DESCRIPTION
**Pull request by Ryaong792**
used the lab examples

### CODE REVIEW

**Comment by Ryaong792** [comment]
fist of 5......1

File: lib/diagnostic.js [hunk 1]
```diff
@@ -9,27 +9,51 @@
 const normalizeWords = function (/* */) {
   /* your response here */
 }
+let words = [paragraph.split(" ")]
+let normalizeWords = [];
+for (let i = 0); i < words.lenght; i++) {
```
Comment (benjimelito) [inline lib/diagnostic.js:14]: Careful on spelling!

File: lib/diagnostic.js [hunk 2]
```diff
@@ -9,27 +9,51 @@
...
+      uniqueWordsAsKeys[normalizeWords[i]++]
+    } else {
+      uniqueWordsAsKeys[normalizeWords[i]]=1;
+    }
+  }
+for (let key in uniqueWordsAsKeys) {
+  if (uniqueWordsAsKeys[key] === 1) {
+    uniqueWords.push(key)
+  }
+}
```
Comment (benjimelito) [inline lib/diagnostic.js:55]: Make sure your functions return something!

**Comment by benjimelito** [comment]
Let me know if you need some 1:1 time during workshop on Thursday to go over some of this. Also, please try to get the PR in under 9:30am

**Pull request closed without merging**